test/screenshotter/test.html

```diff
@@ -31,8 +31,8 @@
     <span id="math"></span>
     <span id="post"></span>
     <script type="text/javascript">
-      function load_fonts(callback) {
-        if (!document.fonts || !document.fonts.load) {
+      function load_fonts_and_images(callback) {
+        if (!document.fonts || !document.fonts.load || !document.images || !document.images.load) {
```

Comment:
```suggestion
        if (!document.fonts || !document.fonts.load) {
```
`document.images` is available on all browsers and there is no `document.images.load`.